ntoskrnl/include/internal/tag.h

```diff
@@ -59,7 +59,6 @@
 #define TAG_IO_TIMER            'MTOI'
 #define TAG_DRIVER              'RVRD'
 #define TAG_DRIVER_EXTENSION    'EVRD'
-#define TAG_SYSB                'BSYS'
```

Comment:
Note, while here:
`TAG_SYSB` was at line 96 too: ok.
Line 87 `TAG_SYS_BUF` too is `'BSYS'`: that "looks" wrong...

Reply:
Indeed, but I didn't want to modify all the places that use `TAG_SYS_BUF` (or alternatively, `TAG_SYSB`) to all use the same name.

Reply:
I can understand that and that is not what I was thinking about:
* if the 2 values should be different, change 1+ of them.
* if they are intended to be identical, define one tag with the other, to make the relation explicit.
Or at least add code comments.